src/Workleap.Extensions.Mongo.Abstractions/TypeExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Reflection;
+
+namespace Workleap.Extensions.Mongo;
+
+internal static class TypeExtensions
```

Comment:
Should it be named MongoTypeExtensions to avoid it becoming a "ramasse-tout" and give a cohesive purpose?